pytorch_lightning/utilities/seed.py

```diff
@@ -53,7 +50,7 @@ def seed_everything(seed: Optional[int] = None) -> int:
         rank_zero_warn(f"{seed} is not in bounds, numpy accepts from {min_seed_value} to {max_seed_value}")
         seed = _select_seed_randomly(min_seed_value, max_seed_value)
 
-    log.info(f"Global seed set to {seed}")
+    rank_zero_info(f"Global seed set to {seed}")
```

Comment:
I think this was done on purpose to make sure the seed is set properly on all ranks. @SeanNaren.

```suggestion
        # using `log.info` instead of `rank_zero_info`, so users can verify the seed is properly set with DDP.
        log.info(f"Global seed set to {seed}")
```

Reply:
hmm... ok if that's the case I'll close this PR, lmk Sean!

Reply:
then it looks like no change, right?

Reply:
Yes. Thomas is correct.

You can use this PR to add a comment explaining why it is done for all ranks

Reply:
agreed with @carmocca - a comment here will be useful